Documentation/cmdref/cilium-agent.md

```diff
@@ -161,6 +161,7 @@ cilium-agent [flags]
       --enable-recorder                                           Enable BPF datapath pcap recorder
       --enable-route-mtu-for-cni-chaining                         Enable route MTU for pod netns when CNI chaining is used
       --enable-sctp                                               Enable SCTP support (beta)
+      --enable-service-protocol-differentiation                   TODO (default true)
```

Comment:
nit, naming bikeshedding 

`--enable-service-protocol-awareness   Enable support for service protocol awareness (TCP, UDP, SCTP) ` ??

Reply:
I think I'll keep the current one as it's also in line with the original gh issue naming